pusher/src/Controller/IoSocketController.ts

```diff
@@ -236,6 +245,8 @@ export class IoSocketController {
                     const websocketExtensions = req.getHeader("sec-websocket-extensions");
                     const IPAddress = req.getHeader("x-forwarded-for");
 
+                    adminApi.setLocale(req.getHeader("accept-language"));
```

Comment:
This is possibly dangerous.

If there is an "async" call between `setLocale` and a method of `adminApi`, there is a risk that the locale will be overidden by the locale of another user.
Don't forget we are in Node, not in PHP. So the `adminApi` object is indeed shared by all users.

I think we should instead explicitly pass the language in parameter of all the methods of `adminApi`.